csdating.py

```diff
@@ -19,3 +19,26 @@ async  def sendhelp(self,message):
         msg = msg + "\n-->: Damit werden 3 Nachrichten mit den Zeiten 19 Uhr,20 Uhr und 21 Uhr schon mit Reactions gesendet!"
         await message.channel.send(msg)
         
+
+    async def reaction(self,payload):
+        #Get the Full message data 
+        message = await self.guilds[0].channels[2].fetch_message(payload.message_id)
+        #iterate over the reaction
+        for reaction in message.reactions:
+            #Full team accepted. (6 because of the Bot)
+            if reaction.emoji == "👍🏻" and reaction.count == 6:
+                #Create a list of the users that reacted to the message
+                users = await reaction.users().flatten()
+                #iterate over the users
+                for user in users:
+                    msg = ("Hi **%s!**\n" % mentionUser(user))
+                    msg = msg +("Du hast ein Date um **%s** mit:\n" % (message.content))
+                    #mention the teammates for user
+                    for teammate in users:
+                        #dont mention the bot and the user himself
+                        if teammate.name == "Rias" or teammate.name == user.name:
```

Comment:
The bot's name is only Rias on our server, not in general. 

Even if nobody else is currently - or ever will be - using our discord bot, I would like you to tweak it into the general case.  (not asap, as we are not in a rush)